docker/Dockerfile

```diff
@@ -1,6 +1,32 @@
 # Inherit from the Drupal 9 image on Docker Hub.
+FROM drupal:9.5 as geos
+# Install git and unzip (needed to build PHP GEOS and needed by Composer).
+# RUN apt-get update \
+#   && apt-get install -y git
+
+ARG PHP_GEOS_VERSION=e77d5a16abbf89a59d947d1fe49381a944762c9d
```

Comment:
Would it be ok to use master branch?
I've checked and that commit has been merged

Reply:
I think my preference would be to keep this as-is for now. As far as I can tell, the `master` branch does not add any new features or fix any bugs, it looks to be just development related changes, so unpinning may introduce more risk than benefit at this time. Ideally we could use a tagged release, but since they have not tagged anything since 1.0.0 in 2017 I'm not sure we can expect new releases. This bears further discussion, though, so maybe it should be a separate PR. That would also make it easier for us to link to independently, if we want to make an official request for a new release from the `php-geos` maintainers.